everyvoice/utils/__init__.py

```diff
@@ -416,3 +416,13 @@ def n_times(n: int) -> str:
     if n == 2:
         return "twice"
     return f"{n} times"
+
+
+def pydantic_validation_error_shortener(e) -> str:
+    """
+    Helper function to shorten pydantic's ValidationError message to only show the first and last extra inputs.
+    """
+    a = str(e).splitlines()
+    if len(a) > 6:
+        return "\n".join(a[:4] + ["..."] + a[-3:])
```

Comment:
Mini détail: should be `> 7`, so that we don't get a 7-line log printed with a spurious `...` in the middle.